portfolio/.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
You don't need to check in this file. Please remove it and instead create a `.gitignore` file to exclude it.